### ISSUE TITLE
Warning: array_replace(): Argument #2 is not an array

### ISSUE THREAD

**Issue by grekpg** [KleeGroup/Recaptcha-bundle-symfony#4]
```
in vendor/kleegroup/google-recaptcha-bundle/RequestMethod/CurlPost.php (line 71)
            CURLOPT_HEADER => false,            CURLOPT_RETURNTRANSFER => true,            CURLOPT_SSL_VERIFYPEER => true        );        $this->curl->setoptArray($handle, array_replace($options , $this->options));        $response = $this->curl->exec($handle);        $this->curl->close($handle);                return $response;    }
```

**Comment by gstolarz-euvic** [comment]
@grekpg 
Confirmed, also it fails on mine installation.

**Comment by BerglindMargo** [comment]
Hi. I'm having the same issue. Did you end up finding a solution for it?

**Comment by t-veron** [comment]
Fixed in [1.0.3 ](https://github.com/KleeGroup/Recaptcha-bundle-symfony/tree/1.0.3)

**Comment by BerglindMargo** [comment]
> Fixed in [1.0.3 ](https://github.com/KleeGroup/Recaptcha-bundle-symfony/tree/1.0.3)

Yay! It works perfectly now. Thank you very much!
